src/quo/components/list_items/dapp/view.cljs

```diff
@@ -1,22 +1,22 @@
 (ns quo.components.list-items.dapp.view
   (:require
-    [quo.components.icon :as icons]
     [quo.components.list-items.dapp.style :as style]
     [quo.components.markdown.text :as text]
-    [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
     [react-native.fast-image :as fast-image]))
 
 (defn view
-  [{:keys [dapp action on-press on-press-icon] :as props}]
+  [{:keys [dapp on-press right-component] :as props}]
```

Comment:
why are we passing in a component here? the design system only shows one variant of button? 
Does it need to reflect the alternatives better? 
Is the right hand side actually generic? 🤔

Reply:
imo it's better to encapsulate all of the styles in the design system that we activate via configuration then handling with dynamic components unless really needed 👍

Reply:
This dapp component can also be used in wallet, there is a `dapps` tab here where it comes without anything. In addition, there is also a variation of it with "..." icon instead of disconnect button.